_releaser/aws.go

```diff
@@ -84,6 +96,189 @@ func (s *AwsLambdaInvokeCmd) Run() error {
 	return nil
 }
 
+type AwsCloudfrontUpdateCmd struct {
+	Region                string `kong:"name='region',env='AWS_REGION'"`
+	Function              string `kong:"name='lambda-function',env='AWS_LAMBDA_FUNCTION'"`
+	FunctionFile          string `kong:"name='lambda-function-file',env='AWS_LAMBDA_FUNCTION_FILE'"`
+	CloudfrontID          string `kong:"name='cloudfront-id',env='AWS_CLOUDFRONT_ID'"`
+	RedirectsJSON         string `kong:"name='redirects-json',env='REDIRECTS_JSON'"`
+	RedirectsPrefixesJSON string `kong:"name='redirects-prefixes-json',env='REDIRECTS_PREFIXES_JSON'"`
+}
+
+func (s *AwsCloudfrontUpdateCmd) Run() error {
+	var err error
+	ver := time.Now().Format("2006-01-02-15:04:05")
```

Comment:
Does this require the custom format? Was curious if any of the default RFCxx formats would do; https://github.com/golang/go/blob/2c46cc8b8997f4f5cdb7766e4e2bdf8e57f67c76/src/time/format.go#L106-L112

Reply:
Looks like it's only used for `Description` (so something we defined ourselves) and in logs, so perhaps good to just use one of the defaults. (does it need an explicit `.UTC()`? not sure), e.g.;

```go
ver := time.Now().UTC().Format(time.RFC3339)
```

Reply:
Yes it's only for lambda version description. Sgtm to simply use the RFC.